docs/database/submarine.sql

```diff
@@ -219,3 +219,32 @@ CREATE TABLE `job` (
   `update_time` datetime default NULL COMMENT 'last update time',
   PRIMARY KEY  (`id`)
   ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+  
+-- ----------------------------
+-- Table structure for metric
+-- ----------------------------
+DROP TABLE IF EXISTS `metric`;
+CREATE TABLE `metric` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `metric_key` varchar(190) NOT NULL COMMENT 'Metric key: `String` (limit 190 characters). Part of *Primary Key* for ``metrics`` table.',
+  `value` float NOT NULL COMMENT 'Metric value: `Float`. Defined as *Non-null* in schema.',
+  `worker_index` varchar(32) NOT NULL COMMENT 'Metric worker_index: `String` (limit 32 characters). Part of *Primary Key* for\r\n    ``metrics`` table.',
+  `timestamp` bigint(20) NOT NULL COMMENT 'Timestamp recorded for this metric entry: `BigInteger`. Part of *Primary Key* for   ``metrics`` table.',
+  `step` bigint(11) NOT NULL COMMENT 'Step recorded for this metric entry: `BigInteger`.',
+  `is_nan` int(11) NOT NULL COMMENT 'True if the value is in fact NaN.',
+  `job_name` varchar(32) NOT NULL COMMENT 'JOB NAME to which this metric belongs to: Part of *Primary Key* for ``metrics`` table.',
+  PRIMARY KEY  (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- ----------------------------
+-- Table structure for param
+-- ----------------------------
+DROP TABLE IF EXISTS `param`;
+CREATE TABLE `param` (
```

Comment:
```suggestion
DROP TABLE IF EXISTS `params`;
CREATE TABLE `params` (
```
pysubmarine tracking API will log parameters to the table named `params`. so here we need to change to `params`